docs/src/pages/components/autocomplete.mdx

```diff
@@ -134,16 +152,17 @@ props.
 ```jsx live
 <div style={{ padding: '2rem 1rem' }}>
     <AutoComplete
+        browserAutoComplete={false}
         dataSource={[
             {
                 label: 'A Custom Results Template',
-                id: 1,
+                id: 'list-item-1',
                 listItemType: 'title',
             },
-            { label: 'Item 1', id: 2, someBoolean: true, isLink:true },
-            { label: 'Item 3', id: 3, someBoolean: true },
-            { label: 'Item 4', id: 4, someBoolean: true },
-            { label: 'Item 5', id: 5, someBoolean: true },
+            { label: 'I\'m a Link!', id: 'list-item-2', isLink: true, },
+            { label: 'Another List Item', id: 'list-item-3', },
+            { label: 'And Another', id: 'list-item-4', },
+            { label: '...You Get The Idea', id: 'list-item-5', },
```

Comment:
Also I think the labels are a little confusing to me. At first it seemed to me like you were saying that only the first results was a Custom Results Template and the others were not. I think keeping these unbolded and listed as `Item , Item 2, etc...` like it was before seems better to me. 🤷